packages/class/types/Request.d.ts

```diff
@@ -8,6 +8,10 @@ export type RequestCreateParams = {
   createAnswer?: RequestAnswerFactory;
 };
 
+export type UploadField = {
```

Comment:
What's the difference with `object`?

Reply:
after update to latest TS, `object` does not passed